client/X11/xf_window.c

```diff
@@ -425,6 +475,17 @@ void xf_ResizeDesktopWindow(xfContext* xfc, xfWindow* window, int width, int hei
 		}
 #endif
 
+		/* Allows the window to resize larger by 1 pixel - so we can
+		 * fullscreen the window with no worries about window manager disallowing based
+		 * on size parameters
+		 */
+		if (can_resize)
+		{
+			size_hints->width_inc = size_hints->height_inc = 1;
+			size_hints->max_width = xfc->width + 1;
+			size_hints->max_height = xfc->height + 1;
```

Comment:
@nfedera I realized today that I might have originally hit this with xfwm4 actually. So, there is a possibility metacity does not require it. I know that one of these two WMs have a problem if not both. I used to run xfwm4 on centos5, but now run metacity on centos6.